java/code/src/com/suse/manager/webui/utils/LoginHelper.java

```diff
@@ -453,4 +456,20 @@ public static User loginUser(String username, String password, List<String> erro
         }
         return user;
     }
+
+    /**
+     * Validate the available disk space using an external script.
+     * @return a string representing the severity level.
+     */
+    public static String validateDiskSpaceAvailability() {
+        final DiskCheck diskCheck = new DiskCheck();
+
+        final DiskCheckSeverity diskCheckSeverity = diskCheck.executeDiskCheck();
+        if (diskCheckSeverity == DiskCheckSeverity.UNDEFINED) {
+            // do not warn if the check was not performed
```

Comment:
is there a reason why we don't just show if there is an issue with the diskspace check? Personally i wouldn't want to ui to pretend everything is fine when in reality it may not have known for some time.

Reply:
My reasoning was that since this type of error should not occur in the first place and since the check happens at every login I would not expect this to fail consistently, thus I did not want to warn immediately the user. But you are right and this behaviour can hide a real issue. I will add a message for this case as well. Maybe this should be just an info saying "Unable to validate the disk space availability. Please review your configuration if this problem persists."?

Reply:
If its something that should not happen or only rarely then i don't see a problem with having the message. My only fear would have been if its something that happens sporadically and fixes it self. As for the the message i think its ok maybe instead of the configuration part point the user to the logs where there will be more information from the exception we log.